WcaOnRails/lib/cutoff.rb

```diff
@@ -1,15 +1,20 @@
 # frozen_string_literal: true
 
 class Cutoff
-  attr_accessor :numberOfAttempts, :attemptValue, :event
-  def initialize(numberOfAttempts: nil, attemptValue: nil, event: nil)
-    self.numberOfAttempts = numberOfAttempts
-    self.attemptValue = attemptValue
+  include ActiveModel::Validations
+
+  attr_accessor :number_of_attempts, :attempt_result, :event
+  validates :number_of_attempts, numericality: { only_integer: true }
+  validates :attempt_result, numericality: { only_integer: true }
+
+  def initialize(number_of_attempts: nil, attempt_result: nil, event: nil)
+    self.number_of_attempts = number_of_attempts
+    self.attempt_result = attempt_result
     self.event = event
   end
 
   def to_wcif
-    { "numberOfAttempts" => self.numberOfAttempts, "attemptValue" => self.attemptValue }
+    { "numberOfAttempts" => self.number_of_attempts, "attemptResult" => self.attempt_result }
```

Comment:
I agree there is some potential to DRY up our `to_wcif` functions, but I don't think this PR is the right place to handle that. I've created #1640 to track doing this this latter.